app/src/main/java/com/matmoongi/network/GsonConverter.kt

```diff
@@ -0,0 +1,7 @@
+package com.matmoongi.network
+
+import retrofit2.converter.gson.GsonConverterFactory
+
+object GsonConverter {
+    val gSonConverter: GsonConverterFactory = GsonConverterFactory.create()
```

Comment:
```suggestion
object GsonConverter {
    val gsonConverter: GsonConverterFactory = GsonConverterFactory.create()
```
`Gson` 이기 때문에 소문자가 될 때는 `gson`이 맞습니다.

Reply:
gson이 한단어인걸 망각해버렸습니다. 수정하겠습니다.